<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="AbsenMasuk">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Absen Masuk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AbsenKeluar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Absen Keluar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
